elephant/test/test_spectral.py

```diff
@@ -158,6 +158,138 @@ def test_welch_psd_multidim_input(self):
         self.assertTrue(np.all(psd_neo_1dim == psd_neo[0]))
 
 
+class MultitaperPSDTestCase(unittest.TestCase):
```

Comment:
Add tests for hierarchical parameters.

Reply:
Added. See https://github.com/NeuralEnsemble/elephant/pull/417/commits/787485f809e71da44fac1482a65505738115478e.